### PR TITLE
Add Tutorial: Install TinyCP on a Hetzner Cloud Server

### PR DESCRIPTION
**Pull request by luizoseas**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Luiz O. Melo Marques luizoseasmm@gmail.com

### CODE REVIEW

File: tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/install-tinycp-on-a-hetzner-cloud-server"
+slug: "Install TinyCP on a Hetzner Cloud Server"
```
Comment (rook59) [inline tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md:4]:
```suggestion
slug: "install-tinycp-on-a-hetzner-cloud-server"
```

File: tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+date: "2020-11-23"
+title: "Install TinyCP on a Hetzner Cloud Server"
+short_description: "This tutorial helps you to install TINYCP on your Ubuntu server."
+tags: ["TinyCP", "Control Panel Server"]
+author: "Luiz O. Melo Marques"
+author_link: "https://github.com/luizoseas"
+author_img: "https://avatars1.githubusercontent.com/u/13169093?s=460&u=b1ca04b31e7bdfd01e62f8e8db42b96b8046cea9&v=4"
```
Comment (rook59) [inline tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md:11]:
```suggestion
author_img: "https://avatars1.githubusercontent.com/u/13169093"
```

File: tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  ```
+  5º Create a new password.
+  ```Shell 
+   New password:
+  ```
+  6º Confirm your new password.
+  ```Shell 
+   Retype new password:
+  ```
+  You have done your remote access on a server on the Hetzner Cloud.
```
Comment (rook59) [inline tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md:79]: I guess you can remove that part and add a "VPS with root access" or similar as prerequisites. This tutorial works on any Ubuntu server.

File: tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+  Before installing TinyCP, it is recommended to update ubuntu packages.
+  ```Shell 
+   apt update && apt upgrade -y
```
Comment (rook59) [inline tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md:83]: You need `sudo` here when you use sudo below.

File: tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+  1º You need to install some packages, run the command:
+  ```Shell 
+   sudo apt install gnupg ca-certificates
+  ```
+  2ºAdd the TinyCP package to your repository, run the commands
+  ```Shell 
+   sudo apt-key adv --fetch-keys http://repos.tinycp.com/ubuntu/conf/gpg.key
+  ```
+  ```Shell 
+   sudo echo "deb http://repos.tinycp.com/ubuntu all main" | sudo tee /etc/apt/sources.list.d/tinycp.list
```
Comment (rook59) [inline tutorials/install-tinycp-on-a-hetzner-cloud-server/01.en.md:98]: echo does not require sudo, only tee does

File: tutorials/install-tinycp-on-a-hetzner-cloud-server/02.pt-br.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/install-tinycp-on-a-hetzner-cloud-server"
+slug: "Instalação TinyCP na Hetzner Cloud Server"
```
Comment (rook59) [inline tutorials/install-tinycp-on-a-hetzner-cloud-server/02.pt-br.md:4]:
```suggestion
slug: "install-tinycp-on-a-hetzner-cloud-server"
```

File: tutorials/install-tinycp-on-a-hetzner-cloud-server/02.pt-br.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/install-tinycp-on-a-hetzner-cloud-server"
```
Comment (rook59) [inline tutorials/install-tinycp-on-a-hetzner-cloud-server/02.pt-br.md:3]:
```suggestion
path: "/tutorials/install-tinycp-on-a-hetzner-cloud-server/pt-br"
```

**Review by rook59 — CHANGES_REQUESTED** [review]
Thank you for your submission. Please take a look at my comments. The comments on the english version apply to both languages.

**Review by luizoseas — COMMENTED** [review]
Changes made

**Review by rook59 — APPROVED** [review]

**Pull request merged**